main.rb

```diff
@@ -48,6 +49,9 @@ def main
 
     options_list(app, menu_option)
     puts ''
+    app.save_books
+    app.save_person
+    app.save_rentals
```

Comment:
- [ ] according to the requirements you should save all the data (books, people, and rentals) into JSON files on exit from your app. because you're calling the methods inside the loop here you are saving the data every time the loop runs which is right after the user makes a choice. please put this right after the loop so it only runs when the loop is broken.

Reply:
This issue is well resolved! ✔️